quill-finagle-mysql/src/main/scala/io/getquill/FinagleMysqlContext.scala

```diff
@@ -28,15 +28,16 @@ import io.getquill.util.LoadConfig
 import io.getquill.util.Messages.fail
 
 class FinagleMysqlContext[N <: NamingStrategy](
```

Comment:
Would it be possible to create a new constructor (`def this`) instead of using a separate parameter group + default param? I think it'd be clearer.